core/templates/pages/admin-page/roles-tab/admin-roles-tab.directive.ts

```diff
@@ -142,9 +142,9 @@ angular.module('oppia').directive('adminRolesTab', [
             formResponse.category, formResponse.username,
             formResponse.languageCode
           ).then(() => {
-            ctrl.setStatusMessage(
-              'Successfully added "' + formResponse.username + '" as ' +
-              formResponse.category + ' reviewer.');
+            ctrl.setStatusMessage(              
```

Comment:
Do we need to change the message on line 139 as well?

Reply:
I will do this as soon as I am able to run it in my local machine in order to see the output!